checks/grouping.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from astropy.table import Table
+from astropy.table import Table
+from astropy.coordinates import Angle, SkyCoord
+from gammapy.data import DataStore
+from gammapy.data import ObservationGroupAxis
+from gammapy.data import ObservationGroups
+from gammapy.datasets import gammapy_extra
+import pylab as pt
+pt.ion()
+
+dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
+data_store = DataStore.from_dir(dir)
+Observation_Table=data_store.obs_table
+alt = Angle([0, 30, 60, 90], 'degree')
+az = Angle([-90, 90, 270], 'degree')
+ntels = np.array([3, 4])
+list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
+                       ObservationGroupAxis('AZ', az, 'bin_edges'),
+                       ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
+obs_groups = ObservationGroups(list_obs_group_axis)
+obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
```

Comment:
This is fine for me to modify the script make_table that I already have!! And we will see after how to generalize this!!
For the moment I am working on the grouping implementation. there will be a lot of modifications but I will try to do a PR before the end of the day to be sure that I am on the good path regarding our discussion!!
Le 21 janv. 2016 à 11:43, Christoph Deil <notifications@github.com> a écrit :

> In checks/grouping.py:
> 
> > +from gammapy.data import ObservationGroups
> > +from gammapy.datasets import gammapy_extra
> > +import pylab as pt
> > +pt.ion()
> > +
> > +dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
> > +data_store = DataStore.from_dir(dir)
> > +Observation_Table=data_store.obs_table
> > +alt = Angle([0, 30, 60, 90], 'degree')
> > +az = Angle([-90, 90, 270], 'degree')
> > +ntels = np.array([3, 4])
> > +list_obs_group_axis = [ObservationGroupAxis('ALT', alt, 'bin_edges'),
> > +                       ObservationGroupAxis('AZ', az, 'bin_edges'),
> > +                       ObservationGroupAxis('N_TELS', ntels, 'bin_values')]
> > +obs_groups = ObservationGroups(list_obs_group_axis)
> > +obs_table_grouped = obs_groups.group_observation_table(Observation_Table)
> For HAP-HD I'm using this script to generate the observation table:
> https://bitbucket.org/hess_software/hddst/src/master/scripts/hap-data-fits-export?fileviewer=file-view-default#hap-data-fits-export-582
> 
> We can / probably should move that into a Gammapy command line tool, but that's a day of work.
> Should we start working on that solution now, or do you want to continue with the script you have (I gather you're using a modified version of the make_table.py script from here, right?)?
> 
> —
> Reply to this email directly or view it on GitHub.
>